src/packages/data-type/workspace/data-type-workspace.context.ts

```diff
@@ -283,6 +284,14 @@ export class UmbDataTypeWorkspaceContext
 			eventContext.dispatchEvent(event);
 		} else {
 			await this.repository.save(this.#currentData.value);
+
+			const actionEventContext = await this.getContext(UMB_ACTION_EVENT_CONTEXT);
```

Comment:
Why do we need to dispatch an event? who is the consumer of those events? .. :-) I'm thinking out loud here.